solutions/devsprint-pedro-menezes-1/DeliveryAppChallenge/Modules/Product/Scenes/GabrielCastro/Coordinator/Coordinator.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+
+
+struct Coordinator {
+    
+   
+    
+}
```

Comment:
pode excluir o arquivo, e quando for cria-lo lembra de cria como `GCLoginCoordinator`